apps/frontend/src/app/OrderElement/OrderElement.tsx

```diff
@@ -0,0 +1,73 @@
+import { Grid, styled, TextField, Typography } from '@mui/material';
+import { Box } from '@mui/system';
+import { OrderModel, OrderStatus } from '@pasnik/api/data-transfer';
+import { Fragment } from 'react';
+import { Link } from 'react-router-dom';
+
+export default function OrderElement(props: {
+  order: OrderModel;
+  index: number;
+}) {
+  const { order, index } = props;
+  const Tile = styled(Box)(({ theme }) => ({
+    ...theme.typography.body2,
+    elevation: 24,
+  }));
+  const routeChange = (menuUrl: string) => {
+    window.open(menuUrl, '_blank');
+  };
+
+  const StyledTextField = styled(TextField)`
+    .MuiOutlinedInput-input {
+      cursor: pointer;
+    }
+  `;
+  return (
+    <Fragment key={index}>
+      <Grid container direction="row">
+        <Grid item xs={9} direction="column">
+          <Tile>
+            <Typography variant="h5" color="initial" display="block">
+              <Link to={`/order/${order.id}`}>{order.from}</Link>
+            </Typography>
+            <Typography variant="h6" color="initial" display="block">
+              <span
+                onClick={(event) => {
```

Comment:
here the inline function is harmless but a good habit would be to extract and wrap it with useCallback